### PR TITLE
Cknieling/rgb based opacity maps arnold

### PR DESCRIPTION
**Pull request by chr11115**
- support rgb based opacity map when using the arnold whader

### CODE REVIEW

File: src/serlio/materials/MaterialUtils.cpp [hunk 1]
```diff
@@ -193,4 +197,14 @@ std::filesystem::path getStingrayShaderPath() {
 	return sfxFile;
 }
 
+bool textureHasAlphaChannel(std::wstring path) {
+	const prt::AttributeMap* textureMetadata = prt::createTextureMetadata((FILE_PREFIX + path).c_str());
```
Comment (mistafunk) [inline src/serlio/materials/MaterialUtils.cpp:201]: mem leak! all return values of prt functions starting with `createXYZ` need manual releasing (through smart ptr of course)

**Review by mistafunk — CHANGES_REQUESTED** [review]

File: src/serlio/materials/MaterialUtils.cpp (hunk 1 above)
Comment (chr11115) [reply src/serlio/materials/MaterialUtils.cpp]: fixed.

[the remark this answers is not in the document]
Comment (chr11115) [reply src/serlio/materials/MaterialUtils.cpp]: fixed.

**Review by mistafunk — APPROVED** [review]
LGTM, feel free to merge.

**Pull request merged**